reset.yml

```diff
@@ -17,7 +17,6 @@
 - hosts: proxmox
   gather_facts: true
   become: yes
-  remote_user: "{{ proxmox_lxc_ssh_user }}"
```

Comment:
why was this removed?

Reply:
It is my understanding this doesn't do anything, per the other CR. We removed this from site.yml, so I updated this to match. I also removed my definition for proxmox_lxc_ssh_user because the root default is fine, and this caused the reset to fail because this variable wasn't defined.

https://github.com/techno-tim/k3s-ansible/pull/263